src/main/java/org/killbill/billing/payment/api/Payment.java

```diff
@@ -84,4 +84,9 @@
      * @return the list of attempts on that payment
      */
     List<PaymentTransaction> getTransactions();
+
+    /**
+     * @return the next Scheduled payment attempt
+     */
+    PaymentAttempt getNextScheduledPaymentAttempt();
```

Comment:
This should probably be `List<PaymentAttempt> getPaymentAttempts();`

Note that retries are implemented as part of the payment attempt framework, but the concept of attempt is more generic and is associated with payment control plugins (http://docs.killbill.io/0.16/userguide_payment.html#_payment_control_plugins).